SYCL/lit.cfg.py

```diff
@@ -281,6 +281,20 @@
 if find_executable('sycl-ls'):
     config.available_features.add('sycl-ls')
 
+llvm_spirv_path = find_executable("llvm-spirv")
+if llvm_spirv_path:
+  lit_config.note("Found llvm-spirv")
+  config.substitutions.append( ('%LLVM_SPIRV', os.path.realpath(llvm_spirv_path)) )
```

Comment:
please use format similar to other substitutions (small letters)

Reply:
Fixed.